src/syscheckd/main.c

```diff
@@ -114,7 +114,7 @@ int main(int argc, char **argv)
         merror(RCONFIG_ERROR, SYSCHECK, cfg);
         syscheck.disabled = 1;
     } else if ((r == 1) || (syscheck.disabled == 1)) {
-        if (syscheck.directories == NULL || syscheck.directories[0] == NULL) {
+        if (syscheck.directories == NULL || syscheck.directories->first_node == NULL) {
```

Comment:
`syscheck.directories->first_node` should be accessed by calling `OSList_GetFirstNode` just to be consistent.